syntax/forth.vim

```diff
@@ -222,9 +222,18 @@ syn match forthTodo contained "\<\%(TODO\|FIXME\|XXX\)\%(\>\|:\@=\)"
 " highlighting for numbers and booleans otherwise it has no effect.
 syn region forthComment start='\<\%(0\|FALSE\)\s\+\[IF]' end='\<\[ENDIF]' end='\<\[THEN]' contains=forthTodo
 
-syn region forthComment start='\<(\>' end=')\>' contains=@Spell,forthTodo,forthSpaceError
+if get(g:, "forth_no_comment_fold", 0)
```

Comment:
Is there any particular reason why one would want a variable for this?

Reply:
Yes it's a bit of a de facto standard - https://github.com/vim/vim/blob/81b8bf5b4a33552c610dc2ea743ac2698a16aef7/runtime/syntax/c.vim#L226

It makes more sense if there's other folding capabilities for the file type.  I can remove it if you like.

Reply:
Probably it does no harm to leave it.